openedx_events/learning/signals.py

```diff
@@ -175,3 +176,14 @@
         "notification_data": UserNotificationData,
     }
 )
+
+# .. event_type: org.openedx.learning.course.created.v1
+# .. event_name: COURSE_CREATED
+# .. event_description: emitted when a course is created
+# .. event_data: CourseData
+COURSE_CREATED = OpenEdxPublicSignal(
+    event_type="org.openedx.learning.course.created.v1",
+    data={
+        "course": CourseData,
+    }
+)
```

Comment:
@rpenido Could you move this signal to [content_authoring/signals.py](https://github.com/openedx/openedx-events/blob/0f6e1ed6481e79f99c593e980d8977741405e2b7/openedx_events/content_authoring/signals.py) instead?